### DISCUSSION TITLE
OSError: ./libsqcloud.so: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by Kelvin-Data** [sqlitecloud/discussions#6] [Bug Report] [answered]
I have installed sqlitecloud.
When I import from sqlitecloud.client import SqliteCloudClient, SqliteCloudAccount
following error incurred: 
[/usr/lib/python3.10/ctypes/__init__.py](https://localhost:8080/#) in __init__(self, name, mode, handle, use_errno, use_last_error, winmode)
    372 
    373         if handle is None:
--> 374             self._handle = _dlopen(self._name, mode)
    375         else:

    376             self._handle = handle

OSError: ./libsqcloud.so: cannot open shared object file: No such file or directory

How to fix it?

**Comment by marcobambini** [comment, marked as answer]
@Kelvin-Data, we have updated the Python SDK, which no longer requires compiling C.
Please follow the new instructions because installing now should be much easier.
Feel free to let me know if you need any help.
